### ISSUE TITLE
documentation for mutex APIs don't mention priority inheritance

### ISSUE THREAD

**Issue by rgrover** [ARMmbed/pal#38]
If multiple tasks with varying priorities wait on a mutex, does the platform port need to implement some kind of priority inheritance? this is important to get clarification on.

**Comment by teetak01** [comment]
Closing the issue as inactive and due to the repository being deprecated.

Please take a look at our current offering at https://cloud.mbed.com/docs/current. The actively maintained version of this component can be found in http://github.com/armmbed/mbed-cloud-client.

Please open a new issue if this is still valid. We strongly encourage you to migrate to the latest product. New issues in this repository will most likely remain unfixed.
